apps/docs/docs/learn/05-theming/03-creating-themes.mdx

```diff
@@ -46,12 +46,12 @@ export const dracula = stylex.createTheme(colors, {
 
 A “theme” is a style object similar to the ones created with `stylex.create()`.
 They can be applied to an element using `stylex.props()` to override variables
-for that element and all its descendents.
+for that element and all its descendants.
 
 ```tsx title="components/MyComponent.js"
 import * as stylex from '@stylexjs/stylex';
-import {colors, spacing} from '../tokens.styles';
-import {dracula} from '../themes';
+import { colors, spacing } from '../tokens.styles';
+import { dracula } from '../themes';
```

Comment:
Change these back as well because we want the examples to be consistent across all pages.

Reply:
What if I update every example or I should leave it?